modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/java/com/liferay/multi/factor/authentication/fido2/web/internal/checker/FIDO2BrowserSetupMFAChecker.java

```diff
@@ -191,6 +184,18 @@ public boolean isBrowserVerified(
 		return false;
 	}
 
+	public void removeCredentialEntry(long credentialEntryId) {
+		try {
+			_mfaFIDO2CredentialEntryLocalService.deleteMFAFIDO2CredentialEntry(
+				credentialEntryId);
+		}
+		catch (PortalException portalException) {
+			if (_log.isWarnEnabled()) {
+				_log.warn(portalException, portalException);
+			}
+		}
+	}
+
```

Comment:
it looks like we are not checking anywere if the user is entitled to remove the credentialEntryId, what if a user tampers with the form and forges a credentialEntryId, is that a feasible scenario?
we should put a check in place to match current user with the credentialEntryId or add a permission check?